common-react/@dbeaver/react-data-grid/src/renderers/OrderButton.tsx

```diff
@@ -0,0 +1,29 @@
+import { IconButton, IconSortAsc, IconSortDesc, IconSortUnknown } from '@dbeaver/ui-kit';
+
+interface OrderButtonProps {
+  attributePosition: number;
+  sortState?: 'asc' | 'desc' | null;
+  onSort: (attributePosition: number, multiple?: boolean) => void;
+  tabIndex?: number;
+  ref: React.Ref<HTMLButtonElement>;
+}
+
+export function OrderButton({ attributePosition, sortState, onSort, tabIndex, ref }: OrderButtonProps) {
+  const handleClick = (e: React.MouseEvent<HTMLButtonElement>) => {
+    onSort(attributePosition, e.ctrlKey || e.metaKey);
+  };
```

Comment:
please prefere to use `function handleClick` instead of arrow functions in such cases